src/lib.rs

```diff
@@ -1797,4 +1839,51 @@ mod tests {
             const D = 8;
         }
     }
+
+    #[test]
+    fn test_iter() {
+        bitflags! {
+            struct Flags: u32 {
+                const ONE  = 0b001;
+                const TWO  = 0b010;
+                const THREE = 0b100;
```

Comment:
Would it be possible to add a cfg'd-out flag to this as well just to make sure we handle those? Something like:

```rust
#[cfg(windows)]
const FOUR_WIN = 0b1000;
#[cfg(unix)]
const FOUR_UNIX = 0b10000;
```

and then use those matching cfgs to check that we iterate over those flags when they're available, but don't when they're not?